apisix/schema_def.lua

```diff
@@ -389,6 +389,10 @@ local upstream_schema = {
             description = "discovery type",
             type = "string",
         },
+        nacos_namespace_id = {
```

Comment:
I think it would be better if we move the `nacos_namespace_id` into `discovery_args`, like:
```
        discovery_args = {
            type = "object",
            properties = {
                namespace_id = {
                    description = "namespace id",
                    type = "string",
                },
            }
        },
```

This change will make the schema tidy.

Reply:
Yes, the would be better！